packages/commonwealth/client/scripts/views/components/component_kit/cw_tabs.tsx

```diff
@@ -15,7 +15,7 @@ type TabStyleAttrs = {
 };
 
 type TabAttrs = {
-  label: string;
+  label: string | m.Vnode;
```

Comment:
Updating this to take an element so that I can render the number of threads beside the label